<launch>

  <arg name="cam_image_topic" default="camera/image_raw" />
  <arg name="cam_info_topic" default="camera/camera_info" />

  <node name="ar_track_alvar"
    pkg="ar_track_alvar"
    type="individualMarkersNoKinect"
    output="screen">
    <rosparam command="load" file="$(find leo_example_follow_ar_tag)/config/alvar.yaml" />

    <remap from="camera_image" to="$(arg cam_image_topic)" />
    <remap from="camera_info" to="$(arg cam_info_topic)" />
  </node>

  <node name="follow_ar_tag"
    pkg="leo_examples"
    type="follow_ar_tag.py">
    <rosparam command="load" file="$(find leo_example_follow_ar_tag)/config/follow_ar_tag.yaml" />
  </node>

</launch>